src/Misc.pm

```diff
@@ -3937,22 +3937,23 @@ sub useTeleport {
 	# 11 Mar 2010 - instead of name, use nameID, names can be different for different servers
 	my $item;
 	if ($use_lvl == 1) { #Fly Wing
-		if (!$config{teleportAuto_item1}) {
-			$item = $char->inventory->getByNameID(601); # Fly Wing
-			unless ($item) { $item = $char->inventory->getByNameID(12323); } # Novice Fly Wing
-			unless ($item) { $item = $char->inventory->getByNameID(23280); } # Beginner's Fly Wing
-		} else {
+		if ($config{teleportAuto_item1}) {
 			$item = $char->inventory->getByName($config{teleportAuto_item1});
 			$item = $char->inventory->getByNameID($config{teleportAuto_item1}) if (!($item) && $config{teleportAuto_item1} =~ /^\d{3,}$/);
+		} unless ($item) {
+			$item = $char->inventory->getByNameID(23280); # Beginner's Fly Wing
+			unless ($item) { $item = $char->inventory->getByNameID(12323); } # Novice Fly Wing
+			unless ($item) { $item = $char->inventory->getByNameID(601); } # Fly Wing
```

Comment:
what about
```perl 
if ($config{teleportAuto_item1}) {
	$item = $char->inventory->getByName($config{teleportAuto_item1});
	$item = $char->inventory->getByNameID($config{teleportAuto_item1}) if (!($item) && $config{teleportAuto_item1} =~ /^\d{3,}$/);
} 
$item = $char->inventory->getByNameID(23280) unless $item; # Beginner's Fly Wing
$item = $char->inventory->getByNameID(12323) unless $item; # Novice Fly Wing
$item = $char->inventory->getByNameID(601) unless $item;     # Fly Wing
```

Reply:
@alisonrag

Reply:
I will fix in some hours :)

Reply:
fixed at:
https://github.com/OpenKore/openkore/pull/3460/commits/14b3ae12107a53cd2ff329c698d301cb3e83bfdc